main.tf

```diff
@@ -1602,6 +1636,7 @@ resource "aws_cloudwatch_event_rule" "ecs_spot_interruption" {
   ],
   "detail": {
     "stopCode": [
+      "SpotInterruption",
```

Comment:
Do we want interruptions to be forwarded for tasks running in a service? The service takes care of this notice by trying to launch new capacity.

One similarity: We show those spot interruptions for EC2 instances no matter if they run in an ASG with Capacity Rebalancing or not.

Reply:
Good question. Maybe we should stick to what we have for ASG?